recipe/meta.yaml

```diff
@@ -72,6 +72,8 @@ requirements:
     #  - the host compiler STL is one to be used.
     #  - it is used by cling's runtime compatibility check.
     - {{ compiler('cxx') }}  # [linux]
+    # TODO: This should match the build version instead of being fixed
+    - libcxx=4  # [osx]
```

Comment:
Thanks!

I don't quite get how this is not already a "strong" run export of `{{ compiler('cxx') }}` on osx.

Reply:
I think `libcxx` is a dependency, but only pinned to being newer than the build time version. As we're using the headers we need to pin it to being the same major (minor?) version.

Reply:
Maybe a `{{ pin_compatible('libcxx') }}`?

With a comment saying that we have a tighter runtime compatibility contraint than the one specified in the run_exports of the cxx compiler?

Although I can't find where this run-exports is set.

(Note: I am fine merging as-is).

Reply:
Yeah something like that should work but I don't have time to test it at the moment.

If you want to edit this PR go ahead else it can stay as a `TODO`.

Reply:
:+1: thanks for fixing this.